apidoc/Titanium/UI/ListSection.yml

```diff
@@ -23,13 +23,20 @@ properties:
   - name: footerView
     summary: View to use for this section footer.
     description: |
-        Using this property and `footerTitle` together is not supported. Use one or the other. 
+        Using this property and `footerTitle` together is not supported. Use one or the other.
         In Alloy you can specify this property with a `<FooterView>` child element of a `<ListSection>`
         element (see Examples).
     type: Titanium.UI.View
     since: 3.2.0
     platforms: [android, iphone, ipad]
 
+  - name: itemCount
+    summary: Returns the item count of the section, also incorporating the search filter if active.
```

Comment:
Getting the filtered count was the point of this PR. Otherwise you have no way to get the real (visible) count. You always can use getItems().length to get the whole item count even if you filter the list.

Reply:
Well, my point here is that iOS does not return the filtered count. It returns the total items (search text does not change this value). I assume iOS' undocumented `itemCount` API was made as an optimization since a `ListView` can have a lot of rows and it's wasteful to fetch a JavaScript array of items via `getItems()` to simply obtain its length.

Perhaps it would be best to create a new API to clearly indicate what's being fetched. Such as `filteredItemCount`. And also maybe`filteredItems` or `filteredIndexes` to obtain an array of which rows were inclusively filtered-in.

Is knowing the filtered item count by itself useful? I'm curious what your use-case is.

Reply:
+1 for `filteredItemCount` and `itemCount`. We currently do not have the space to do the iOS parity, so this PR may need more community work.